.github/ISSUE_TEMPLATE/product-submission-form.yml

```
@@ -0,0 +1,78 @@
+---
+name: Submit a Product
+description: Submit a product to the database
+title: "[Product Submission]: <vendor> - <product>"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        For questions about choice for status, please see the information below.
+
+        - Unknown - Status unknown. Default choice.
+        - Affected - Reported to be affected by CVE-2021-44228.
+        - Not Affected - Reported to NOT be affected by CVE-2021-44228 and no
+          further action necessary.
+        - Fixed - Patch and/or mitigations available (see provided links).
+        - Under Investigation - Vendor investigating status.
+  - type: input
+    id: product-vendor
+    attributes:
+      label: Product vendor
+      description: Who is the vendor for the product?
+    validations:
+      required: true
+  - type: input
+    id: product-name
+    attributes:
+      label: Product name
+      description: What is the product?
+    validations:
+      required: true
+  - type: input
+    id: product-versions
+    attributes:
+      label: Product version(s)
+      description: What version(s) of the product are affected?
+    validations:
+      required: true
+  - type: dropdown
+    id: product-status
+    attributes:
+      label: Product status
+      description: What is the current status of the affected product?
+      options:
+        - Unknown
+        - Affected
+        - Not Affected
+        - Fixed
+        - Under Investigation
+    validations:
+      required: true
+  - type: dropdown
+    id: product-updated
+    attributes:
+      label: Product update available
+      description: Is there an update available for the product?
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: input
+    id: product-update-link
+    attributes:
+      label: Product update link
+      description: If an update is available where can it be found?
```

I think this warrants a comma, but feel free to disagree.
```suggestion
      description: If an update is available, where can it be found?
```

---

I think this wording is slightly better, but I'm fine either way:
```suggestion
      description: If an update is available where can it be found?
      description: Where can an update be found, if one is available?
```

---

Please see c5c6c68.